iterreg/utils.py

```diff
@@ -24,7 +24,7 @@ def plot_legend_apart(ax, figname, ncol=None):
 
 
 @njit
-def shrink(u, tau):
+def shrink(u, tau, factor = 1.):
```

Comment:
this is not pep8 and surprising to the user, I think you can do:
`shrink(u, alpha, lc)` (and work a bit on the docstring)